### DISCUSSION TITLE
@formily/react与@formily/vue的FormTab组件的schema数据格式不统一

### DISCUSSION THREAD

**Discussion by borina555** [alibaba/formily#2967] [Q&A 2.x] [answered]
-         tab1: {

          type: 'void',
          'x-component': 'FormTab.TabPane',
          'x-component-props': {
            **tab: 'A1',**
          },
          properties: {
            aaa: {
              type: 'string',
              title: 'AAA',
              'x-decorator': 'FormItem',
              required: true,
              'x-component': 'Input',
            },
          },
        },
上面是@formily/react

- const schema = {
  type: 'object',
  properties: {
    collapse: {
      type: 'void',
      'x-component': 'FormTab',
      'x-component-props': {
        formTab: '{{formTab}}',
      },
      properties: {
        tab1: {
          type: 'void',
          'x-component': 'FormTab.TabPane',
          'x-component-props': {
            **label: 'A1',**
          },
          properties: {
            aaa: {
              type: 'string',
              title: 'AAA',
              'x-decorator': 'FormItem',
              required: true,
              'x-component': 'Input',
            },
          },

@formily/element的tab的schema数据结构

**Comment by janryWang** [comment, marked as answer]
组件都不一样，没必要保持一致

**Comment by borina555** [comment]
按我的理解是，底层的schema数据格式统一，不依赖于外部的ui组件，这样还好吧



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年3月22日(周二) 下午4:01
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [alibaba/formily] @***@***.***/vue的FormTab组件的schema数据格式不统一 (Discussion #2967)




 
组件都不一样，没必要保持一致

**Reply by janryWang** [reply]
不好，需要formily来抹平差异，这会导致用户不知道看哪个文档
